src/tests/src/solver/modeler/api/CMakeLists.txt

```diff
@@ -1,2 +1,24 @@
-include(LinearProblemWithOrtools.cmake)
-include(LinearProblemBuilder.cmake)
\ No newline at end of file
+set(EXECUTABLE_NAME test-modeler-linear-problem)
```

Comment:
_Renaming_ 

`set(EXECUTABLE_NAME test-modeler-linear-problem)` : 
as, in the end, we are building one executable for modeler API (see path to this cmake file), 
we could call this executable **tests-for-modeler-api**

Reply:
done